### ISSUE TITLE
[1.20.1] You cannot have any origins when you are offline

### ISSUE THREAD

**Issue by Reniel80** [EdwinMindcraft/origins-architectury#401]
I'm using a custom origins addon. when i'm online sometimes some layers are not visible in the selection windows (this is the first bug), but when i'm offline.. you cannot select any origins... selection windows is missing and when you press the [ o ] key to see the info... it says "You cannot have any origins"
![no origins](https://github.com/EdwinMindcraft/origins-architectury/assets/98614051/26bed66e-dbbc-40f9-bb3d-40486b6ef1fb)

**Comment by ChrysanthCow** [comment]
I don't think this is really a bug to do with being offline, it's more of a bug to do with the registry system, which will be fixed next update.

**Comment by DanfeUran** [comment]
Hi, idk if you fixed it in last update but I have same problem

**Comment by DanfeUran** [comment]
mby this can help 
![image](https://github.com/EdwinMindcraft/origins-architectury/assets/163349574/7e68af73-ce65-45e8-adf1-d46f61118c07)

**Comment by DanfeUran** [comment]
> I don't think this is really a bug to do with being offline, it's more of a bug to do with the registry system, which will be fixed next update.

here is my log btw

[log for you ;).txt](https://github.com/EdwinMindcraft/origins-architectury/files/14628318/log.for.you.txt)

**Comment by DanfeUran** [comment]
So I relouded in to the same world and its working
